src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -72,6 +72,12 @@ Cold and warm start are Mobile Vitals, which you can learn about in the [full do
 
 ## Slow and Frozen Frames
 
+<Alert level="info" title="Important">
+
+We recommend updating to at least [7.18.1](https://github.com/getsentry/sentry-cocoa/releases/tag/7.11.0), because the SDK might report a too high percentage of slow frames.
```

Comment:
```suggestion
We recommend updating to at least [7.18.1](https://github.com/getsentry/sentry-cocoa/releases/tag/7.11.0), because the SDK might report too high a percentage of slow frames.
```